components/KiboOrderHistory.vue

```diff
@@ -0,0 +1,260 @@
+<template>
+  <div>
+    <div v-show="activePage != 'Order History'" class="order-history-title">
+      {{ activePage === "My Account" ? "Order History" : "View Order Details" }}
```

Comment:
I think we should use localization here for "Order History"  and "View Order Details"